### PR TITLE
Add missing geps to mkdocs

### LINKED ISSUE #2112: Some GEPs not discoverable in sidebar
Some GEPs, such as [GEP-1686](https://gateway-api.sigs.k8s.io/geps/gep-1686/) for example, do not appear to be listed in the Enhancement Proposal list in the sidebar on https://gateway-api.sigs.k8s.io/geps/overview/.  Is this meant to be a complete list of GEPs or is there some other more authoritative way to discover all of the GEPs?

### PR DESCRIPTION
**Pull request by levikobi**
**What type of PR is this?**

**What this PR does / why we need it**:
/kind documentation

**Which issue(s) this PR fixes**:

Fixes #2112

**Does this PR introduce a user-facing change?**:

```release-note
Added missing GEPs.
```

### CODE REVIEW

**Review by adleong — APPROVED** [review]
Thank you!

**Review by robscott — COMMENTED** [review]
Thanks @levikobi! One small thing but otherwise LGTM.

/ok-to-test
/approve

File: mkdocs.yml [hunk 1]
```diff
@@ -85,13 +85,17 @@ nav:
         - geps/gep-1282.md
         - geps/gep-1619.md
         - geps/gep-1897.md
+        - geps/gep-1876.md
```
[the remark this answers is not in the document]
Comment (levikobi) [reply mkdocs.yml:88]: Thanks man! good catch

**Review by shaneutt — APPROVED** [review]
/approve
/lgtm

File: mkdocs.yml [hunk 2]
```diff
@@ -85,13 +85,17 @@ nav:
...
       - Prototyping:
         - geps/gep-1709.md
       - Experimental:
         - geps/gep-1748.md
         - geps/gep-1016.md
         - geps/gep-957.md
         - geps/gep-713.md
+      - Implementable:
+        - geps/gep-1426.md
+        - geps/gep-1686.md
```
Comment (frankbu) [inline mkdocs.yml:98]: Shouldn't Implementable come before Experimental, like in the flow chart?
```suggestion
      - Implementable:
        - geps/gep-1426.md
        - geps/gep-1686.md
      - Experimental:
        - geps/gep-1748.md
        - geps/gep-1016.md
        - geps/gep-957.md
        - geps/gep-713.md
```

Comment (robscott) [reply mkdocs.yml:98]: Good catch, thanks @frankbu!

**Review by shaneutt — APPROVED** [review]
/lgtm

**Pull request merged**